### ISSUE TITLE
Thermostat with 2 devices

### ISSUE THREAD

**Issue by vasid** [actions-on-google/smart-home-nodejs#142]
Hello, 
Right now I am working on a google home app for thermostats.

I have a problem while trying to control 2 thermostats while using the commands "Turn up the temperature by 5" or "Turn down the temperature by 5".
If the thermostats temperatures are different, on QUERY we return 2 devices and on EXECUTE we get only one device, so the temperature is set only to that device (same device for turn up and turn down).
If the thermostats temperatures are the same, we get 2 devices on QUERY  and 2 devices on EXECUTE, the it works as it should(for turn up and turn down).

Also, if I try the commands "Set the temperature to 25" or "Set the heating to 25", it works for both devices on different or the same temperatures.

Can anyone help me with this issues? Or if there is a workaround to let me know?

Thank you,
Vasile Dit

**Comment by Fleker** [comment]
Have you configured the rooms for each device, and say the room name that you want to adjust?

When you are getting an EXECUTE intent for just one particular device, is that device consistently the same one?

What do you expect the behavior should be?

**Comment by vasid** [comment]
Hello,
Both thermostats have rooms assigned, and the command "Turn up the temperature by 5" was a broadcast command, for both of them, "Turn up the bedroom temperature by 5" works without problems.

The strange thing that I've noticed that the broadcast command("Turn up the temperature by 5") works if the temperatures are the same on both thermostats, but doesn't not work if they are different.

From my tests I've noticed that on EXECUTE i get always the same device.

The expected behavior should be that both thermostats get their temperature increased with 5 degrees.
